git-subhistory.sh

```diff
@@ -267,18 +273,27 @@ subhistory_assimilate () {
 		revs_to_rewrite=ASSIMILATE_HEAD
 	fi
 
-	git filter-branch \
+        commit_count=$(git rev-list --count $revs_to_rewrite)
+
+        if test $commit_count -ne 0
+        then
```

Comment:
these are indented 1 space more than they should be; compare with the `fi` a few lines above